.schema/openapi/patches/hydra.yaml

```diff
@@ -1,31 +1,15 @@
-# These patches are needed, because we merge the Kratos spec with the Hydra spec (and others) to create the 
+# These patches are needed, because we merge the Kratos spec with the Hydra spec (and others) to create the
 # Ory Client spec.
 # Hydra defines "OAuth2Client" (and the other schemas mentioned here) as well, which is where Kratos' "OAuth2Client"
 # comes from originally. It is generated by our SDK generator. This leads to a clash and some generators can not
 # deal with this. So we rename these fields to be lowercase, so that they can be merged in the spec for the `client`
 # generation.
 # Also see https://github.com/ory/sdk/issues/226
-- op: copy
-  from: /components/schemas/OAuth2Client
-  path: /components/schemas/oAuth2Client
-- op: remove
-  path: /components/schemas/OAuth2Client
-- op: replace
-  path: /components/schemas/OAuth2LoginRequest/properties/client/$ref
-  value: '#/components/schemas/oAuth2Client'
-
-- op: copy
-  from: /components/schemas/OAuth2ConsentRequestOpenIDConnectContext
-  path: /components/schemas/oAuth2ConsentRequestOpenIDConnectContext
-- op: remove
-  path: /components/schemas/OAuth2ConsentRequestOpenIDConnectContext
-- op: replace
-  path: /components/schemas/OAuth2LoginRequest/properties/oidc_context/$ref
-  value: '#/components/schemas/oAuth2ConsentRequestOpenIDConnectContext'
-
-- op: copy
-  from: /components/schemas/OAuth2LoginRequest
+- op: add
   path: /components/schemas/oAuth2LoginRequest
+  value:
```

Comment:
@jonas-jonas looks like this is the only thing we need

Reply:
Can you please check whether that fixes the problem in the sdk generator?

Reply:
No, that causes the same errors, because there are still two `oauth2Client`s present in the spec. We somehow need to get rid of one of them or distinguish them with more than just casing.